test/karma/spec/features.js

```diff
@@ -0,0 +1,22 @@
+import OktaAuth from '@okta/okta-auth-js';
+
+describe('features', function() {
+
+  // Assumes our test browser is genuinely capable of these features.
+  describe('expected results in test browser', () => {
+
+    it('isPopupPostMessageSupported', () => {
+      expect(OktaAuth.features.isPopupPostMessageSupported()).toBe(true);
+    });
+
+    it('isTokenVerifySupported', () => {
+      expect(OktaAuth.features.isTokenVerifySupported()).toBe(true);
+    });
+
+    it('isPKCESupported', () => {
```

Comment:
Can we also add a failure case?

Reply:
I added several failure cases, but they are on the jest side. see `spec/features.js`